api/envoy/extensions/wasm/v3/wasm.proto

```diff
@@ -17,6 +18,23 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // [#protodoc-title: Wasm]
 // [#extension: envoy.bootstrap.wasm]
 
+// If there is a fatal error on the VM (e.g. exception, abort()), then the policy will be applied.
+enum FailurePolicy {
+  // No policy is specified. The default policy will be used. The default policy is ``FAIL_RELOAD``.
+  UNSPECIFIED = 0;
+
+  // All plugins associated with the VM will be reloaded then new requests are processed. This
+  // make sense when the VM failure is caused by runtime exception, abort(), or panic().
+  FAIL_RELOAD = 1;
+
+  // All plugins associated with the VM will return an HTTP 503 error.
+  FAIL_CLOSED = 2;
+
+  // All plugins associated with the VM will be ignored and the filter chain will continue. This
+  // is makes sense when the plugin is optional.
+  FAIL_IGNORE = 3;
```

Comment:
Should we call this mode `FAIL_OPEN` which is the common terminology used on ext authz etc?

Reply:
No strong preference. I personally prefer the `ignore` because it tell the users the failure will be ignored and the filter chain will continue. But `FAIL_OPEN` is also OK to me if the api-shepherd prefer that. :)

Reply:
FWIW, I also think that `FAIL_OPEN` is a little more intuitive here.

Reply:
Got it.